src/manager.js

```diff
@@ -654,7 +654,13 @@ class Manager extends EventEmitter {
 
     const { rows } = await this.db.executeSql(sql)
 
-    return rows.at(0) || null
+    return rows.at(0) || {
```

Comment:
I think this result should be merged into the result of `getQueueCache(name)` so it can overwrite cached values found.  This would also remove the need of introducing a new type.